src/core/components/eventEmitter.js

```diff
@@ -83,39 +89,46 @@ export default class EventEmitter {
         data: e.payload.data,
       };
       this.listenerManager.announceObjects(announce);
+      this._announce('objects', announce, announce.channel, announce.subscription);
```

Comment:
I never thought about this, but why we announce about incompletely processed object?

Reply:
we can have listener handler  like 
```javascript
{
....
objects: (o)=>  {...},

user/space: (e)=> {...}
}
```
It was there previously as well.. One is general `object` with all kind of update/addition of entity.

user/space is with context terms.

Reply:
Ok